core/src/main/java/com/es/core/model/phone/PhoneExtractor.java

```diff
@@ -0,0 +1,39 @@
+package com.es.core.model.phone;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.ResultSetExtractor;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class PhoneExtractor implements ResultSetExtractor<List<Phone>> {
```

Comment:
`PhoneExtractor` - class name is a little confusing. Other developer might think that this class suppose to extract only one phone from ResultSet. So I would call it PhoneListExtractor or PhonesExtractor